From f4b722391de129f7854c0fddeb8be8836f8ed122 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Wed, 18 Jan 2012 10:38:14 +0100
Subject: [PATCH 20/52] qemu-io: fix aio help texts

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1326883126-22053-21-git-send-email-armbru@redhat.com>
Patchwork-id: 36589
O-Subject: [RHEL-6.3 PATCH qemu-kvm 20/52] qemu-io: fix aio help texts
Bugzilla: 758194
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>

From: Christoph Hellwig <hch@lst.de>

Fix a few typos in the help texts for the various aio commands.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>
(cherry picked from commit e432cef9161685ea3dbb3f31a90820e53f74041e)
Signed-off-by: Markus Armbruster <armbru@redhat.com>
---
 qemu-io.c |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-io.c |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/qemu-io.c b/qemu-io.c
index b1484f3..f8bbd4c 100644
--- a/qemu-io.c
+++ b/qemu-io.c
@@ -902,8 +902,8 @@ aio_read_help(void)
 "\n"
 " Reads a segment of the currently open file, optionally dumping it to the\n"
 " standard output stream (with -v option) for subsequent inspection.\n"
-" The read is performed asynchronously and should the aio_flush command \n"
-" should be used to ensure all outstanding aio requests have been completed\n"
+" The read is performed asynchronously and the aio_flush command must be\n"
+" used to ensure all outstanding aio requests have been completed\n"
 " -C, -- report statistics in a machine parsable format\n"
 " -P, -- use a pattern to verify read data\n"
 " -v, -- dump buffer to standard output\n"
@@ -1003,8 +1003,8 @@ aio_write_help(void)
 "\n"
 " Writes into a segment of the currently open file, using a buffer\n"
 " filled with a set pattern (0xcdcdcdcd).\n"
-" The write is performed asynchronously and should the aio_flush command \n"
-" should be used to ensure all outstanding aio requests have been completed\n"
+" The write is performed asynchronously and the aio_flush command must be\n"
+" used to ensure all outstanding aio requests have been completed\n"
 " -P, -- use different pattern to fill file\n"
 " -C, -- report statistics in a machine parsable format\n"
 " -q, -- quite mode, do not show I/O statistics\n"
@@ -1095,7 +1095,7 @@ aio_flush_f(int argc, char **argv)
 static const cmdinfo_t aio_flush_cmd = {
 	.name		= "aio_flush",
 	.cfunc		= aio_flush_f,
-	.oneline	= "completes all outstanding aio requets"
+	.oneline	= "completes all outstanding aio requests"
 };
 
 static int
-- 
1.7.7.5

